src/forms/Form.react.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+import * as React from "react";
+
+type Props = {|
+  +action: string,
+  +children?: Array<React.Node>,
+  +method: string,
+  +title: string,
+|};
+
+function Form(props: Props): React.Node {

```

Comment:
Do we need to be/should we be using more specific return types for components where possible? For example in this case `function Form(props: Props): React.StatelessFunctionalComponent<Props>`?

Reply:
So `React.StatelessFunctionalComponent<Props>` will not be the correct type here (see attached flow error).  The most specific type I can think of would be `React.Element<"form">` but I'd rather just use React.Node here as that's what `render()` functions usually yield.

Reply:
![flow-error](https://user-images.githubusercontent.com/9311453/38808469-4fe29230-414e-11e8-8a8a-fd664484030b.PNG)

Reply:
Ah yes this makes sense, I'll stick with React.Node for them all